WikiPreview.Fluent.Plugin/WikiPreviewSearchApp.cs

```diff
@@ -57,14 +58,16 @@ public ValueTask<IHandleResult> HandleSearchResult(ISearchResult searchResult)
             if (string.IsNullOrWhiteSpace(displayedName))
                 return new ValueTask<IHandleResult>(new HandleResult(true, false));
 
+            string titleUrl = displayedName.Replace(' ', '_');
```

Comment:
Cast the `ISearchResult` to `WikiSearchResult` and use the `URL` property instead of trying to change the `DisplayedName`